tests/testthat/test_print_anova.R

```diff
@@ -13,7 +13,7 @@ test_that(
     tw_aov1 <- apa_print(aov(yield ~ N * P, npk), es = c("ges"), observed = "N")
     tw_aov2 <- apa_print(aov(yield ~ N * P, npk), es = c("pes"), observed = "N")
     tw_aov3 <- apa_print(aov(yield ~ N * P, npk), es = c("es"), observed = "N")
-    expect_equal(tw_aov1$estimate$N, "$\\hat{\\eta}^2_G = .224$")
+    expect_equal(tw_aov1$estimate$N, "$\\hat{\\eta}^2_G = .218$")
```

Comment:
Hmm, why has this value changed?

Reply:
Ìf I remember correctly, `observed` has not been passed properly before.

Reply:
Think I remember correctly:
```
> afex::aov_4(yield ~ (N*P|block), npk)
Anova Table (Type 3 tests)

Response: yield
  Effect   df   MSE      F  ges p.value
1      N 1, 5 19.70 9.61 * .224    .027
2      P 1, 5 14.28   0.59 .013    .478
3    N:P 1, 5 28.84   0.74 .031    .430
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘+’ 0.1 ‘ ’ 1
> afex::aov_4(yield ~ (N*P|block), npk, observed = "N")
Anova Table (Type 3 tests)

Response: yield
  Effect   df   MSE      F  ges p.value
1      N 1, 5 19.70 9.61 * .218    .027
2      P 1, 5 14.28   0.59 .010    .478
3    N:P 1, 5 28.84   0.74 .025    .430
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘+’ 0.1 ‘ ’ 1
```

Reply:
Hmm, that's not so great...